data.sql

```diff
@@ -2,7 +2,13 @@
 
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES
-	(1,'Agumon', TO_DATE('03/02/2020', 'MM/DD/YYYY'), 0, true, 10.23),
-	(2,'Gabumon', TO_DATE('11/15/2020', 'MM/DD/YYYY'), 2, true, 8),
-	(3,'Pikachu', TO_DATE('01/07/2021', 'MM/DD/YYYY'), 1, false, 15.04),
-	(4,'Devimon', TO_DATE('05/12/2017', 'MM/DD/YYYY'), 5, true, 11);
\ No newline at end of file
+	(5,'Charmander', TO_DATE('02/08/2020', 'MM/DD/YYYY'), 0, false, 11),
+	(6,'Plantmon', TO_DATE('11/15/2021', 'MM/DD/YYYY'), 2, true, 5.7),
+	(7,'Squirtle', TO_DATE('04/02/1993', 'MM/DD/YYYY'), 3, false, 12.13),
+	(8,'Angemon', TO_DATE('06/12/2005', 'MM/DD/YYYY'), 1, true, 45),
```

Comment:
Kindly add the proper weight for animals that have negative weights. The negative weights are intentional, if you do not include them it will hinder the results in your queries.

Note: The query

```sql
UPDATE animals
SET weight = weight * -1
WHERE weight < 0;
```

is meant to correct the negative weights, therefore they need to exist in order for the query to be useful.

For example:
`Charmander` weight should be  `-11.0`
`Plantmon` weight should be `-5.7')`
`Squirtle` weight should be `-12.13`
`Angemon` weight should be `-45.0`